loky/backend/reduction.py

```diff
@@ -80,6 +81,7 @@ def _rebuild_partial(func, args, keywords):
 # global variable to change the pickler behavior
 try:
     import cloudpickle  # noqa: F401
+
     DEFAULT_ENV = "cloudpickle"
 except ImportError:
     # If cloudpickle is not present, fallback to pickle
```

Comment:
Since cloudpickle is hard dependency, I think it's safe to just set ``DEFAULT_ENV = "cloudpickle"``

Reply:
If for some reason, someone installs `loky` without `cloudpickle` (with `--no-deps`), the library would still be working (not the tests thought). So there is a change of behaviour. I feel that this change deserve its own PR.